sceptre/nlpsandbox/config/common/nlpsandboxvpc.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: remote/vpc-mini.yaml
```

Comment:
isn't adding a VPC already in PR #12 ?

Reply:
I marked this PR as a draft because I want to merge #12 first so the diffs are minimal. I've changed the base branch for the PR to simulate a merge. You can take another look now. It should only be the CI workflow.